README.md

```diff
@@ -10,6 +10,10 @@ See http://transition.city/
 
 For Ubuntu users: [complete step-by-step development environment setup procedure](docs/setupDevEnvironmentUbuntu20.04.md)
 
+## User installation
```

Comment:
I would change the title to something like
"Stand alone Desktop installation"

user installation is not really clear at what it is. Could be interpreted as "add user to the platform)